snapcraft.yaml

```diff
@@ -12,20 +12,11 @@ environment:
   PATH: $SNAP/usr/sbin:$SNAP/usr/bin:$SNAP/sbin:$SNAP/bin:$PATH
   LC_ALL: C
   OS_PLACEMENT_CONFIG_DIR: $SNAP/etc/nova/
-  # admin rc (TODO: put this someplace more sensible)
-  OS_PROJECT_DOMAIN_NAME: default
-  OS_USER_DOMAIN_NAME: default
-  OS_PROJECT_NAME: admin
-  OS_USERNAME: admin
-  OS_PASSWORD: keystone
-  OS_AUTH_URL: http://10.20.20.1:5000
-  OS_IDENTITY_API_VERSION: 3
-  OS_IMAGE_API_VERSION: 2
```

Comment:
It feels so good to get rid of all these hard coded values and put them in a place (microstack.rc) that makes sense to somebody hacking on it!